x-pack/plugins/ml/server/routes/modules.ts

```diff
@@ -91,6 +92,59 @@ export function dataRecognizer(
       )
     );
 
+  router.versioned
```

Comment:
Can you add `summary` and `description` props for the new route handler?

Reply:
Added in 9838277f8bb24511c72c3af4da087cbdb52624e7